### ISSUE TITLE
IndexError: list index out of range (2)

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#101]
As per
https://github.com/test-fullautomation/python-jsonpreprocessor/issues/86#issue-1791756024
indices (in square brackets) must be placed outside the curly brackets.

The test case:

```
"list_param" : ["A", "B", "C"],
"val2"       : "${list_param[1]}"
```

produces the following error message:

`list index out of range`

This is misleading.

Expected is something like

`Invalid syntax: Found index inside curly brackets. Indices in square brackets have to be placed outside the curly brackets.`

**Comment by test-fullautomation** [comment]
@namsonx : What is the status of this issue?

**Comment by test-fullautomation** [comment]
Hi @namsonx ,
what is the status of this ticket?
Thank you,
 Thomas

**Comment by namsonx** [comment]
Hello Holger, Hello Thomas,

I pushed new [commit 8015387a3](https://github.com/test-fullautomation/python-jsonpreprocessor/commit/8015387a3019ed3cddafd611b4575bb22d6748af) to stabi branch to check and print out detail error message log.
The error message log in this case is: ```Invalid syntax: Found index inside curly brackets in line '"val2"       : "${list_param[1]}"'. Indices in square brackets have to be placed outside the curly brackets.```

I also updated the [BADCASE] test cases JPP_0350, JPP_0351, JPP_0352 to adapt new message log.

Thank you,
Son

**Comment by test-fullautomation** [comment]
Hi Son,
looks good to me!
Thank you,
 Thomas

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by namsonx** [comment]
merged to develop branch

**Comment by test-fullautomation** [comment]
solved with version 0.10.0
